package.json

```diff
@@ -14,9 +14,11 @@
     "posttest:dev:functional": "bash tests/functional/test.sh stop",
     "test:dev:lint": "node_modules/eslint/bin/eslint.js ./src/scripts/**/*.js",
     "test:stylelint": "node_modules/stylelint/dist/cli.js './src/**/*.scss' --config webpack/.stylelintrc",
-    "dev": "node webpack/dev-server.js & PORT=8000 node start.js",
+    "dev": "env NODE_PATH='./src' PORT=8000 UV_THREADPOOL_SIZE=100 node webpack/webpack-dev-server.js & PORT=8000 node ./bin/server.js",
     "start": "NODE_PATH=\"./src\" node ./start",
-    "build": "node ./node_modules/webpack/bin/webpack.js --config webpack/prod.config.js",
+    "build": "npm run build:client & npm run build:server",
+    "build:server": "babel src -d dist -D",
```

Comment:
you might want to include `--copy-files` to copy over any none js files. 

http://stackoverflow.com/questions/32642685/babel-cli-copy-nonjs-files